iroh-p2p/src/node.rs

```diff
@@ -531,6 +535,19 @@ impl<KeyStorage: Storage> Node<KeyStorage> {
     async fn handle_rpc_message(&mut self, message: RpcMessage) -> Result<bool> {
         // Inbound messages
         match message {
+            RpcMessage::ExternalAddrs(response_channel) => {
+                response_channel
+                    .send(
+                        self.swarm
+                            .external_addresses()
+                            .map(|r| r.addr.clone())
```

Comment:
you can use `.cloned()` instead

Reply:
like this? Or is there a "one step" `clone` + `map`?

```
self.swarm
        .external_addresses()
        .cloned()
        .map(|r| r.addrs)
        .collect()
```

Reply:
nvm, just leave it as it is, I misread